db/migrate/20220308210841_move_user_uploads_to_upload_references.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class MoveUserUploadsToUploadReferences < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      INSERT INTO upload_references(upload_id, target_type, target_id)
+      SELECT upload_id, 'User', user_id
+      FROM user_uploads
+    SQL
+  end
+
+  def down
```

Comment:
Should have `    raise ActiveRecord::IrreversibleMigration` here IMO

Reply:
I do not think so, but I agree that we should have something here. If up copies data from a table to another one, down should copy that data back. In this case, that becomes difficult and unlikely to happen so that's why I left this blank.

Reply:
If it's difficult and unlikely then the migration is irreversible and should be marked as such IMO, we follow this pattern all the time.